Volume/Views/PublicationDetail/PublicationDetailHeader.swift

```diff
@@ -10,17 +10,24 @@ import SwiftUI
 
 struct PublicationDetailHeader: View {
     @EnvironmentObject private var userData: UserData
+    @State private var hasOddNumberOfTaps = false
     private let iconGray = Color(white: 196 / 255)
 
     let publication: Publication
     
+    // Whether the publication is followed at the time this view is displayed
     private var isFollowed: Bool {
         userData.isPublicationFollowed(publication)
     }
 
     private var shoutouts: Int {
         max(publication.shoutouts, userData.shoutoutsCache[publication.id, default: 0])
     }
+    
+    // Takes into account any new user taps of the following button
+    private var isFollowedAdjusted: Bool {
+        isFollowed && !hasOddNumberOfTaps || !isFollowed && hasOddNumberOfTaps
```

Comment:
This is the same as `isFollowed xor hasOddNumberOfTaps`. In Swift, `^` is only defined for ints, but fortunately for bools it is equivalent to `!=`. 

Also, I would put the logic of `isFollowedAdjusted` into `isFollowed` and update the comment accordingly. Specifically, I would say:

```
// Takes into account any new user taps of the following button
private var isFollowed: Bool {
    userData.isPublicationFollowed(publication) != didToggleFollowed
}
```

Do you like that better or no?

Reply:
Bump on this

Reply:
Good point. Waiting for the 3410 logic to start kicking in.